scalafix-tests/output/src/main/scala/test/NoExtendsApp.scala

```diff
@@ -0,0 +1,51 @@
+package test
+
+object Main {
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+trait Something
+object Main2 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main3 extends Something{
+  def main(args: Array[String]) = {
+    println(s"Hello, ${args(0)}")
+    println(s"Hello 2, ${args(0)}")
+  }
+}
+
+object Main4 extends Something
+
+object Main5 extends Something {}
+
+object Main6
+  extends Something
+
+object Main7
+  extends Something
+
+object Main8
+  extends Something
+
+object Main9 extends
+  Something
+
+import scala.{ App => Ppa }
```

Comment:
I wouldn't worry about that, RemoveUnusedImport also creates unused imports in cases like

```scala
import foo.bar
import bar._ // unused import

// after rewrite
import foo.bar // unused import
```

I'm afraid users will need to run scalafix at least twice to be sure all unused imports got removed.